src/shared/defaultAwsContext.ts

```diff
@@ -107,14 +102,9 @@ export class DefaultAwsContext implements AwsContext {
     // resets the context to the indicated profile, saving it into settings
     public async setCredentialAccountId(accountId?: string): Promise<void> {
         this.accountId = accountId
-        await this.settingsConfiguration.writeSetting(accountIdSettingKey, accountId, vscode.ConfigurationTarget.Global)
```

Comment:
Now that accountId has no persistent state, we need to ensure that it is set each time the Extension is activated.

Reply:
✔️